src/string-class.js

```diff
@@ -0,0 +1,165 @@
+/* @file string-class */
+const stringClassExtensions = {
+  /**
+   * hasVowels
+   * Test to see if a string contains vowels
+   * @return {Boolean} true or false
+   */
+  hasVowels() {
+    const vowels = /[aeiou]/gi;
+    return vowels.test(this);
+  },
+
+  /**
+   * toUpper
+   * Converts a string to uppercase
+   * @return {String} uppercase equivalent of the string
```

Comment:
Nitpick: The proper JSDoc return statement is `@returns` instead of `@return` as specified [here](http://usejsdoc.org/tags-returns.html).